src/components/ModelTableList/OwnerGroup.js

```diff
@@ -78,7 +81,10 @@ export default function OwnerGroup({ activeUser }) {
             {
               content: (
                 <a href="#_" className="p-link--soft">
-                  {getStatusValue(model.info, "cloudCredentialTag")}
+                  {getStatusValue(
+                    model.info,
+                    "cloudCredentialTag"
+                  ).toLowerCase()}
```

Comment:
Why do we want to lowercase credentials? Credentials are case sensitive. So if the user copies creds from the dashboard they will fail.

Reply:
Didn't realise that - will remove.